### ISSUE TITLE
Predictions Plugin - init() is not public

### ISSUE THREAD

**Issue by Andrei-C96** [aws-amplify/amplify-swift#3009]
### Describe the bug

A few months ago, in version 2.8.1 (2023-04-17) the Predictions plugin `init()` was made internal in this [PR](https://github.com/aws-amplify/amplify-swift/pull/2855) in order to avoid calling unimplemented APIs until the v2 library will be implemented.

The issue is that when Predictions v2 was merged in version 2.9.0 (2023-04-27) in this [PR](https://github.com/aws-amplify/amplify-swift/pull/2902), the plugin's `init()` remained internal, thus making it impossible to configure Amplify with that plugin.

### Steps To Reproduce

```swift
1. Create an empty Xcode project
2. Follow the instructions in the documentation to add the Predictions plugin: https://docs.amplify.aws/lib/predictions/getting-started/q/platform/ios/
3. At that point, you can see that this line: "try Amplify.add(plugin: AWSPredictionsPlugin())" will produce the following error: "'AWSPredictionsPlugin' initializer is inaccessible due to 'internal' protection level"
```

### Expected behavior

The expected behaviour would be for the app to successfully configure Amplify with the Predictions plugin, instead the app is unable to build due to the internal init.

With a simple edit to that init, the plugin is functional again. In my case I've forked the repo to test this out here: https://github.com/Andrei-C96/amplify-swift/commit/41169789b8c86a18f16981fd03dcf5a662151572

### Amplify Framework Version

2.11.5

### Amplify Categories

Predictions

### Dependency manager

Swift PM

### Swift version

5

### CLI version

10.4.0

### Xcode version

14.3 (14E222b)

### Is this a regression?

No

### Device

iPhone 14 Pro Max - Simulator

### iOS Version

iOS 16.4

**Comment by atierian** [comment]
Thanks for bringing this to our attentions @Andrei-C96. 
The PR to make this public is pending review and we'll work to get it released ASAP. I'll get you updated here.
- https://github.com/aws-amplify/amplify-swift/pull/3011

Thanks again!

**Comment by Andrei-C96** [comment]
That's great news! 😄
Thank you for your fast response @atierian

**Comment by atierian** [comment]
The fix was released in [version 2.11.6](https://github.com/aws-amplify/amplify-swift/releases/tag/2.11.6).
Thanks again for reporting this!

**Comment by Andrei-C96** [comment]
Awesome, I've tested it out and everything works great 👍🏻

I'll go ahead and close this issue since everything is resolved.
Thanks again for your help and for your quick responses!
